configs/server/forseti_conf_server.yaml.sample

```diff
@@ -180,6 +180,15 @@ inventory:
         #    - spanner.googleapis.com/Instance
         #    - sqladmin.googleapis.com/Instance
         #    - storage.googleapis.com/Bucket
+        #    - container.googleapis.com/Cluster
```

Comment:
container.googleapis.com/Cluster is already included above.

Reply:
Removed new entry.